www/js/lib/uiUtil.js

```diff
@@ -415,9 +415,12 @@ define(rqDef, function() {
      * For each contentTheme, a stylesheet must be provided in www/css that is named 'kiwixJS' + contentTheme
      * A rule may additionally be needed in app.css for full implementation of contentTheme
      * 
-     * @param {String} theme The theme to apply (light|dark[_invert|_mwInvert])
+     * @param {String} theme The theme to apply (light|dark[_invert|_mwInvert]|auto[_invert|_mwInvert])
      */
     function applyAppTheme(theme) {
+        var darkPreference = window.matchMedia('(prefers-color-scheme:dark)');
+        var selectedThemeContent = theme.replace(/^[^_]*/, '');
```

Comment:
`selectedThemeContent` is exactly the same code as `contentTheme` below. It looks like redundant code.

Reply:
this variable is to store the content theme the user has chosen as we are gonna change the the theme values for `auto` themes (so these values are not gonna be same). if won't do this those values will be lost. And this variable is making the code more compact instead of using three `if` statements. Also I'm using this for wikimedia help to remove the invalid part from the `light` theme.